com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -83,6 +83,14 @@ gravity_y.type = number
 gravity_y.help = world gravity along y-axis, -10 by default (natural gravity)
 gravity_y.default = -10
 
+velocity_iteration.type = number
+velocity_iteration.help = modify amount of iteration for velocity in Box2D physics
+velocity_iteration.default = 16
```

Comment:
The default is 10, as seen in the physics_2d.cpp.
Always be mindful of backwards compatibility.

Reply:
Will set them to 10 in next PR.